integrations/azure_ai_search/src/haystack_integrations/document_stores/azure_ai_search/filters.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict
+
+from dateutil import parser
+
+from .errors import AzureAISearchDocumentStoreFilterError
+
+LOGICAL_OPERATORS = {"AND": "and", "OR": "or", "NOT": "not"}
```

Comment:
Don't we have enum for these somewhere, please search

Reply:
I found them [here](https://github.com/deepset-ai/haystack/blob/78292422f00592bb0a6b5d58bbbb679f4b8718da/haystack/utils/filters.py#L48) in haystack.utils but thats not what we need. Plus other integrations also define these operators based on requirements.